interceptors/userInterceptor.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken')
+const {HttpError} = require("../helper/httpErrors");
+
+async function setUser(ctx, next) {
```

Comment:
this mindware should be called `auth`, instead of interceptor call this a middleware(folder and files)

Reply:
Folder and files renamed from interceptor to middleware